Pod/Cache/GenericCoreDataStack.swift

```diff
@@ -258,7 +258,7 @@ public class GenericCoreDataStack<CoordinatorType: NSPersistentStoreCoordinator,
             //
             if context.parentContext == mainContext {
                 
-                mainContext.performBlockAndWait({ () -> Void in
+                mainContext.performBlock( { () -> Void in
                     do {
                         try self.mainContext.save()
```

Comment:
Do you need to check whethere the mainContext.hasChanges, before saving? I have seen that done elsewhere.